app/src/main/AndroidManifest.xml

```diff
@@ -5,9 +5,23 @@
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
+        <activity android:name="lt.vilnius.tvarkau.StartActivity" />
+
+        <!-- This strangely named alias is here for backwards compatibility.
+             Renaming activity with LAUNCHER category means that shortcuts created by users
+             on their phones with previous version of the app will disappear or stop working. -->
+        <activity-alias
+            android:name="apps.factory.emiestas.MainActivity"
```

Comment:
As I mentioned, our limited resource is our time. 
Thinking/Discussing/Typing - all this takes our precious time that we can use instead for droping few extra lines of code.

I vote FOR changing the package name as long as doesn't put under risk to loose any of 786 existing users. However, I'm encouraging everyone to keep focusing on our goals and use our time with the greatest benefit to the project.